src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/commands.py

```diff
@@ -112,6 +115,22 @@ def load_command_table(self, _):  # pylint: disable=too-many-statements
         operations_tmpl='azure.cli.command_modules.acr.check_health#{}'
     )
 
+    acr_scope_map_util = CliCommandType(
+        operations_tmpl='azure.cli.command_modules.acr.scope_map#{}',
+        client_factory=cf_acr_scope_maps
+    )
+
+    acr_token_util = CliCommandType(
+        operations_tmpl='azure.cli.command_modules.acr.token#{}',
+        client_factory=cf_acr_tokens
+    )
+
+    acr_token_credential_generate_util = CliCommandType(
+        operations_tmpl='azure.cli.command_modules.acr.token#{}',
+        table_transformer=credential_output_format,
```

Comment:
Does this table transformer map with the parameters for token credentials? If not, you will need a separate one.